background.ts

```diff
@@ -250,6 +252,56 @@ async function checkForRootPrivs() {
   }
 }
 
+async function checkPrerequisites() {
+  const osPlatform = os.platform();
+  let messageId: window.reqMessageId = 'ok';
+
+  switch (osPlatform) {
+  case 'win32': {
+    // Required: Windows 10-1909(build 18363) or newer
+    const winRel = os.release().split('.');
```

Comment:
Note that Windows 11 seems to have this fixed at `22000`. Don't think we care about that right now (we support any version of Windows 11 at the moment), but it's useful to know.